pkg/plugins/balance/balance.go

```diff
@@ -0,0 +1,66 @@
+package balance
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	clusterapiv1 "open-cluster-management.io/api/cluster/v1"
+	clusterapiv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
+	"open-cluster-management.io/placement/pkg/plugins"
+)
+
+const placementLabel = "cluster.open-cluster-management.io/placement"
+
+var _ plugins.Prioritizer = &Balance{}
+
+type Balance struct {
+	meta   metav1.ObjectMeta
+	handle plugins.Handle
+}
+
+func New(handle plugins.Handle) *Balance {
+	return &Balance{handle: handle}
+}
+
+func (b *Balance) Name() string {
+	return "balance"
+}
+
+func (b *Balance) Set(pl *clusterapiv1alpha1.Placement) {
+	b.meta = pl.ObjectMeta
+}
+
+func (b *Balance) Score(ctx context.Context, clusters []*clusterapiv1.ManagedCluster) (map[string]float64, error) {
```

Comment:
Clever implementation.  No weighting of what's on there, but the concept is solid.  I think I prefer integer scoring to float scoring.  Any particular attachment to floats?